src/mesh/simple_meshes.jl

```diff
@@ -1,23 +1,212 @@
+function findline(name, lines)
+    for (i, line) in enumerate(lines)
+        if line == name
+            return i
+        end
+    end
+end
+
+struct MeshOptions
+    grouping::Bool
+    remapGroupName::Bool
+end
+
+"""
+ returns the number of elements in a .msh file of a specified dimension
+"""
+function getNumElements(lines,Dim=2)
+    elem_start = findline("\$Elements",lines) + 1
+    temp_numBlocks,_,_,_ = split(lines[elem_start])
```

Comment:
```suggestion
    temp_numBlocks, _ = split(lines[elem_start])
```
I think the trailing `_` just discards other output arguments.